site/get-data.js

```diff
@@ -0,0 +1,30 @@
+const swagImages = [];
+const fileNames = [];
+const swagList = require('../data.json').map(swag => {
+    // Generate unique filename
+    // @todo: do we want to force jpeg as the image format?
+    // const extension = swag.image.split('.').pop();
```

Comment:
Also, this kind of cut is not really safe, it should rely on mimetype instead

Reply:
True, but from what I can tell that adds quite a bit of complexity to the build system. While there could be a CI check (i.e. make sure the mime types match the extension) the complexity cost isn't worth it in the gulpfile

Reply:
I agree, that's why jpeg was forced

Reply:
We could also manually specify the format in the data file and default to jpeg if it's not provided 🤔